database/src/test/java/info/metadude/android/eventfahrplan/database/extensions/SQLiteDatabaseExtensionsTest.kt

```diff
@@ -0,0 +1,103 @@
+        @Test
+        fun `addTextColumn with NULL`() {
+            db.addTextColumn(tableName = "sessions", columnName = "url", default = "NULL")
+            verifyInvokedOnce(db).execSQL("ALTER TABLE sessions ADD COLUMN url TEXT DEFAULT 'NULL'")
+        }
+
+        @Test
+        fun `addTextColumn with empty string`() {
+            db.addTextColumn(tableName = "sessions", columnName = "url", default = "")
+            verifyInvokedOnce(db).execSQL("ALTER TABLE sessions ADD COLUMN url TEXT DEFAULT ''")
+        }
+
+        @Test
+        fun `addTextColumn with number`() {
+            db.addTextColumn(tableName = "sessions", columnName = "url", default = "0")
+            verifyInvokedOnce(db).execSQL("ALTER TABLE sessions ADD COLUMN url TEXT DEFAULT '0'")
+        }
+
+        @Test
+        fun `addTextColumn with boolean word`() {
+            db.addTextColumn(tableName = "sessions", columnName = "url", default = "true")
+            verifyInvokedOnce(db).execSQL("ALTER TABLE sessions ADD COLUMN url TEXT DEFAULT 'true'")
+        }
+
+        @Test
+        fun `addTextColumn with text`() {
+            db.addTextColumn(tableName = "sessions", columnName = "url", default = "lorem")
+            verifyInvokedOnce(db).execSQL("ALTER TABLE sessions ADD COLUMN url TEXT DEFAULT 'lorem'")
+        }
+
+    }
+
+    @Nested
+    inner class AddIntegerColumn {
+
+        @Test
+        fun `addIntegerColumn with NULL`() {
+            db.addIntegerColumn(tableName = "sessions", columnName = "count", default = "NULL")
+            verifyInvokedOnce(db).execSQL("ALTER TABLE sessions ADD COLUMN count INTEGER DEFAULT NULL")
+        }
+
+        @Test
+        fun `addIntegerColumn with empty string`() {
+            try {
+                db.addIntegerColumn(tableName = "sessions", columnName = "count", default = "")
+            } catch (e: IllegalArgumentException) {
+                assertThat(e.message).isEqualTo("Default value must be an integer string or NULL not \"\".")
+            }
+        }
+
+        @Test
+        fun `addIntegerColumn with number`() {
+            db.addIntegerColumn(tableName = "sessions", columnName = "count", default = "0")
+            verifyInvokedOnce(db).execSQL("ALTER TABLE sessions ADD COLUMN count INTEGER DEFAULT 0")
+        }
+
+        @Test
+        fun `addIntegerColumn with boolean word`() {
+            try {
+                db.addIntegerColumn(tableName = "sessions", columnName = "count", default = "true")
+            } catch (e: NumberFormatException) {
+                assertThat(e.message).isEqualTo("For input string: \"true\"")
+            }
+        }
+
+        @Test
+        fun `addIntegerColumn with text`() {
+            try {
+                db.addIntegerColumn(tableName = "sessions", columnName = "count", default = "lorem")
+            } catch (e: NumberFormatException) {
+                assertThat(e.message).isEqualTo("For input string: \"lorem\"")
+            }
+
+        }
```

Comment:
All of these tests pass if no exception is thrown. 

You could use JUnit5's `assertThrows()` (or switch to [assertk](https://github.com/willowtreeapps/assertk) for proper Kotlin support):

```kotlin
val exception = assertThrows<SpecificException> {
    // do something
}
assertThat(exception).hasMessageThat().isEqualTo("Exception message")
```

Reply:
:heavy_check_mark: Yes, `fail() is missing. But let's try `assertThrows()`. Thanks!